modules/chamber_params/main.tf

```
@@ -24,3 +24,22 @@ resource "aws_ssm_parameter" "ignored_values" {
     ]
   }
 }
+
+data "aws_ssm_parameter" "this" {
+  for_each = aws_ssm_parameter.this
+  name     = "/${lower(var.group)}/${lower(each.key)}"
```

```suggestion
  for_each = aws_ssm_parameter.ignored_values
  name     = each.key
```
Iterate over `ignored_values`. `name` is already given in `aws_ssm_parameter.ignored_values`